.chloggen/failover-added-tests.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
We should omit the change log since this isn't user facing. I'll add the Skip Changelog label.